### PR TITLE
feat(shared-data): add calibration commands to schemaV7

### PR DESCRIPTION
**Pull request by smb2268**
# Overview

This PR adds the `calibration/calibratePipette` and `calibration/moveToLocation` commands and schemaV7 command types
# Changelog

1. Copy all schemaV6 types to a new schemaV7 type folder
2. Create new `calibration.ts` file with `calibration/calibratePipette` and `calibration/moveToLocation` commands
3. Add these commands and to `7-draft.json` with a warning that they shouldn't be used in protocols

# Review requests

Verify that the command type, params, and result align with what was added in https://github.com/Opentrons/opentrons/pull/11482 and https://github.com/Opentrons/opentrons/pull/11478/files

# Risk assessment

Low

### CODE REVIEW

File: shared-data/protocol/schemas/7-draft.json [hunk 1]
```diff
@@ -518,6 +518,23 @@
...
+          {
+            "description": "[This command is for internal use only and should NOT be used in protocols.] Initiate automatic calibration of a pipette.",
+            "type": "object",
+            "required": ["commandType", "params"],
+            "properties": {
+              "key": { "type": "string" },
+              "commandType": { "enum": ["calibration/calibratePipette"] },
+              "params": {
+                "required": ["mount"],
+                "labwareId": {
```
Comment (TamarZanzouri) [inline shared-data/protocol/schemas/7-draft.json]:
```suggestion
                "mount": {
```
Should this be mount?

Comment (smb2268) [reply shared-data/protocol/schemas/7-draft.json]: yep thanks! leftover from a copy/paste

**Review by jerader — APPROVED** [review]
✅

File: shared-data/protocol/types/schemaV7/command/calibration.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  deckSlot: SetupPosition
+}
+
+interface MoveToLocationResult {
+  deckPoint: {
+    x: number
+    y: number
+    z: number
+  }
+  positionId: string
```
Comment (caila-marashaj) [inline shared-data/protocol/types/schemaV7/command/calibration.ts]: I took the `positionId` out of the `MoveToLocationResult` class, so it's just the `deckPoint` that's in here now

File: shared-data/protocol/types/schemaV7/command/calibration.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  mount: PipetteMount
+}
+
+interface CalibratePipetteResult {
+  pipetteOffset: LabwareOffset
+}
+
+interface MoveToLocationParams {
+  pipetteId: string
+  deckSlot: SetupPosition
```
Comment (caila-marashaj) [inline shared-data/protocol/types/schemaV7/command/calibration.ts]: also recently renamed the `deckSlot` param to `location`

**Review by caila-marashaj — DISMISSED** [review]
Just a couple small things in the comments to get this up to date with my changes from today

**Review by TamarZanzouri — APPROVED** [review]
changes lgtm! thank you for addressing my comment :-)

File: shared-data/protocol/types/schemaV7/command/calibration.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+export type CalibrationRunTimeCommand =
+  | CalibratePipetteRunTimeCommand
+  | MoveToLocationRunTimeCommand
+
+export type CalibrationCreateCommand =
+  | CalibratePipetteCreateCommand
+  | MoveToLocationCreateCommand
+
+export const ATTACH_DETACH_POSITION = 'attachOrDetach'
```
Comment (caila-marashaj) [inline shared-data/protocol/types/schemaV7/command/calibration.ts]: not sure if this has to align with the python names of the enum, but if it does, `ATTACH_DETACH_POSITION` will want to be `ATTACH_OR_DETACH`

File: shared-data/protocol/types/schemaV7/command/calibration.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+export const PROBE_POSITION = 'probePosition'
+export type SetupPosition =
```
Comment (caila-marashaj) [inline shared-data/protocol/types/schemaV7/command/calibration.ts]: same thing here, if it needs to be the same, `SetupPosition` will correspond to `CalibrationPosition`

File: shared-data/protocol/types/schemaV7/command/calibration.ts (hunk 4 above)
Comment (smb2268) [reply shared-data/protocol/types/schemaV7/command/calibration.ts]: they technically don't need to align but probably easier to read if they do - just updated!

**Pull request merged**